packages/core/modules-sdk/src/medusa-app.ts

```diff
@@ -84,7 +84,7 @@ export async function loadModules(args: {
     sharedResourcesConfig,
     migrationOnly = false,
     loaderOnly = false,
-    workerMode = "server" as ModuleBootstrapOptions["workerMode"],
+    workerMode = "shared" as ModuleBootstrapOptions["workerMode"],
```

Comment:
q: What happened here?

Reply:
this is supposed to be "shared" by default.
we have already other places setting this as "shared" if not provided, it is just to align with the real default.